[2025-05-14T05:15:48.655+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_ingestion.process_olist_order_reviews_dataset manual__2025-05-14T05:13:11.949842+00:00 [queued]>
[2025-05-14T05:15:48.859+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_ingestion.process_olist_order_reviews_dataset manual__2025-05-14T05:13:11.949842+00:00 [queued]>
[2025-05-14T05:15:48.881+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-14T05:15:49.254+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): process_olist_order_reviews_dataset> on 2025-05-14 05:13:11.949842+00:00
[2025-05-14T05:15:49.380+0000] {standard_task_runner.py:60} INFO - Started process 297 to run task
[2025-05-14T05:15:49.433+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_ingestion', 'process_olist_order_reviews_dataset', 'manual__2025-05-14T05:13:11.949842+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/ecommerce_etl_dag.py', '--cfg-path', '/tmp/tmp7320qxyb']
[2025-05-14T05:15:49.468+0000] {standard_task_runner.py:88} INFO - Job 139: Subtask process_olist_order_reviews_dataset
[2025-05-14T05:15:49.989+0000] {task_command.py:423} INFO - Running <TaskInstance: ecommerce_data_ingestion.process_olist_order_reviews_dataset manual__2025-05-14T05:13:11.949842+00:00 [running]> on host 1e43c0d7b523
[2025-05-14T05:15:50.405+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ecommerce_data_ingestion' AIRFLOW_CTX_TASK_ID='process_olist_order_reviews_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-05-14T05:13:11.949842+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-14T05:13:11.949842+00:00'
[2025-05-14T05:15:50.412+0000] {ecommerce_etl_dag.py:65} INFO - Processing file: /opt/***/data/extracted_data/olist_order_reviews_dataset.csv
[2025-05-14T05:15:50.516+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2025-05-14T05:15:51.118+0000] {ingestion_script.py:328} INFO - Processing olist_order_reviews_dataset.csv...
[2025-05-14T05:17:07.944+0000] {ingestion_script.py:384} INFO - Uploaded 99224 valid records to s3://rj-ecommerce-data-lake/raw/olist_order_reviews_dataset/2025-05-14/olist_order_reviews_dataset.csv
[2025-05-14T05:17:08.682+0000] {python.py:201} INFO - Done. Returned value was: Successfully processed olist_order_reviews_dataset.csv
[2025-05-14T05:17:08.737+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ecommerce_data_ingestion, task_id=process_olist_order_reviews_dataset, execution_date=20250514T051311, start_date=20250514T051548, end_date=20250514T051708
[2025-05-14T05:17:08.806+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-14T05:17:08.839+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
